From 267e1de395ff6bf7275ba0e795d6a74e50989a1a Mon Sep 17 00:00:00 2001
From: He Chuyue <hechuyue@wxiat.com>
Date: Fri, 9 Sep 2022 14:23:37 +0800
Subject: [PATCH] sw64: remove useless enum
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 267e1de395ff6bf7275ba0e795d6a74e50989a1a
Modified-by-SEL: No


Sunway inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I56OLG

--------------------------------

Signed-off-by: He Chuyue <hechuyue@wxiat.com>
Reviewed-by: He Sheng <hesheng@wxiat.com>
Signed-off-by: Gu Zitao <guzitao@wxiat.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/sw_64/kernel/ptrace.c | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/arch/sw_64/kernel/ptrace.c b/arch/sw_64/kernel/ptrace.c
index f3bc1020eaff..e98679d10fae 100644
--- a/arch/sw_64/kernel/ptrace.c
+++ b/arch/sw_64/kernel/ptrace.c
@@ -46,16 +46,6 @@
  * zero have no stack-slot and need to be treated specially (see
  * get_reg/put_reg below).
  */
-enum {
-	REG_R0 = 0,
-	REG_F0 = 32,
-	REG_FPCR = 63,
-	REG_PC = 64,
-	REG_SP = 30,
-	REG_PS = 31,
-	REG_GP = 29
-};
-
 #define R(x)	((size_t) &((struct pt_regs *)0)->x)
 
 short regoffsets[32] = {
-- 
2.33.0

